src/components/PeopleFilters.tsx

```diff
@@ -16,77 +69,102 @@ export const PeopleFilters = () => {
             type="search"
             className="input"
             placeholder="Search"
+            value={query}
+            onChange={(event) => setCurrentQuery(event.target.value)}
           />
 
           <span className="icon is-left">
-            <i className="fas fa-search" aria-hidden="true" />
+            <i
+              className="fas fa-search"
+              aria-hidden="true"
+              onClick={deleteQuery}
+            />
           </span>
         </p>
       </div>
 
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
+            <NavLink
               data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
+              className={selecetedCentury.includes(15)
+                ? 'button mr-1 is-info'
+                : 'button mr-1'}
+              to="#/people?centuries=16"
+              onClick={() => handleCenturySelection(15)}
             >
               16
-            </a>
+            </NavLink>
 
-            <a
+            <NavLink
```

Comment:
Create a separate component for these links. Look, you use the same logic, just different numbers. Create an array with possible centuries and iterate over it.